cypress/e2e/scenario-8-instance-composer.cy.js

```diff
@@ -744,7 +744,7 @@ if (Cypress.env("edition") === "iso") {
       // Expect Canvas to be visible
       cy.get(".canvas").should("be.visible");
 
-      cy.get("main").scrollTo("bottom"); //scroll to the bottom as the error container is clipped in 1500x900 viewport
+      //      cy.get("main").scrollTo("bottom"); //scroll to the bottom as the error container is clipped in 1500x900 viewport
```

Comment:
I had to comment this out to have the tests pass localy in the ui, no idea if we still need it as is or not. Weird that the changes of this pr affected this though...

Reply:
we can remove that as it is no longer required I guess.

It's weird that it was working solution for failing scenario and then it was causing errors :D